index.js

```diff
@@ -0,0 +1,59 @@
+import fs from 'fs';
+
+const data = fs.readFileSync('words_alpha.txt', 'utf8');
+const words = data.split('\r\n');
+const ALPHABET = ['a', 'b', 'c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z'];
+
+function areAlphabetical(words){
+
+    let alphabeticalWords = [];
+    
+    for (let i = 0; i < words.length; i++){
+
+        if (isAlphabetical(words[i])){
+            alphabeticalWords.push(words[i])
+        }
+    }
+
+    return isLongestWord(alphabeticalWords)
+ }
+
+
+function isAlphabetical(word){
+
+    let alphabeticalWord = [];
+    let count = 0; 
+
+    for (let j = 0; j < word.length; j++){
+       alphabeticalWord.push(ALPHABET.indexOf(word[j]))
+    } 
+    
+    for (let i = 0; i < alphabeticalWord.length-1; i++){
+
+        if (alphabeticalWord[i+1] > alphabeticalWord[i]){
+            count += 1
+        }
+    }
+    
+    return count === alphabeticalWord.length-1
+}
+
+
+function isLongestWord(arr){
```

Comment:
functions called isXXX or hasXXX are expected to return booleans.
This function returns the longest word so i would call it getLongestWord.